### ISSUE TITLE
[FEATURE REQ] rTorrent add support for Digest Authentication

### ISSUE THREAD

**Issue by Pingblue** [autobrr/autobrr#571]
Currently only Basic authentication scheme is supported in Autobrr when adding the rtorrent client  it would be nice to see support for those who use Digest authentication.

**Comment by zze0s** [comment]
Hi! We rely on this library [mrobinsn/go-rtorrent](https://github.com/mrobinsn/go-rtorrent) so it's probably best to ask there to start with. They don't properly handle Basic Auth atm either so I doubt they'll want to do anything with Digest Auth.

We might look into forking it and making it fit our needs. We'll see.

**Comment by zze0s** [comment]
We recently improved the handling of Basic Auth for rtorrent in this PR https://github.com/autobrr/autobrr/pull/914 and in the process we forked the library so now we have full control.

I looked into Digest Auth briefly and it seem to be somewhat similar to oauth.

What is your current setup that supports Digest Auth? This is one of those features for sub 1% of users, but if it's easy enough to implement we might do it.
